templates/CSI/cephfs/pod.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ pod_name | default('nginx-pod-cephfs') }}
+spec:
+  containers:
+   - name: web-server
+     image: nginx
```

Comment:
@nehaberry does it make sense to make this configurable?

Reply:
@ebenahar u mean the pod image type ? since its type is web server, it would be better to use only eb server type app pods.

I am open to suggestions here. :) 
One more thing we can do is - we can keep all app pods under a separate directory and create different templates of major ones. Please suggest.

Reply:
@nehaberry good idea, but let's wait with that for now